src/main/java/online/store/book/model/Book.java

```diff
@@ -0,0 +1,29 @@
+package online.store.book.model;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import java.math.BigDecimal;
+import lombok.Data;
+
+@Entity
+@Table(name = "books")
+@Data
```

Comment:
Don't use Data for entities
```suggestion
@Getter
@Setter
```